unpack issue

Hi,

trying to compile tag 6.0.1 (clean build). Got stuck at package installer with following error
`
  UNPACK   installer
find: 'build.OpenELEC-Generic.x86_64-6.0.1/installer*': No such file or directory
scripts/unpack: line 155: build.OpenELEC-Generic.x86_64-6.0.1/installer-1/.openelec-unpack: No such file or directory
make: *** [image] Fehler 1
`
Tried it now two times with a real fresh git clone the second time. Still happening.
Any ideas?

--- a/scripts/unpack
+++ b/scripts/unpack
@@ -94,6 +94,10 @@ if [ -d "$PKG_DIR/sources" ]; then
   cp -PRf $PKG_DIR/sources/* $BUILD/${PKG_NAME}-${PKG_VERSION}
 fi
 
+if [ -z "$PKG_URL" ]; then
+  mkdir -p "${BUILD}/${PKG_NAME}-${PKG_VERSION}"
+fi
+
 if [ "$(type -t post_unpack)" = "function" ]; then
   post_unpack
 fi

